apps/test-site/vite.config.js

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from "vite";
```

Comment:
Is it possible for pagesjs to provide a default vite config if the user doesn't have one? I'd rather not have extra vite configs floating around the studio repo when we're just using the recommended default.

Reply:
If it's quick could we make the change in PagesJS? We don't need to release it right away but I'd like to remove these at a later date